packages/x-date-pickers/src/locales/nlNL.ts

```diff
@@ -70,11 +70,11 @@ const nlNLPickers: Partial<PickersLocaleText> = {
   dateTableLabel: 'kies datum',
 
   // Field section placeholders
-  fieldYearPlaceholder: (params) => 'Y'.repeat(params.digitAmount),
+  fieldYearPlaceholder: (params) => 'J'.repeat(params.digitAmount),
   fieldMonthPlaceholder: (params) => (params.contentType === 'letter' ? 'MMMM' : 'MM'),
   fieldDayPlaceholder: () => 'DD',
   fieldWeekDayPlaceholder: (params) => (params.contentType === 'letter' ? 'EEEE' : 'EE'),
-  fieldHoursPlaceholder: () => 'hh',
+  fieldHoursPlaceholder: () => 'uu',
```

Comment:
Yeah, ok, makes sense: 

<img width="1068" alt="SCR-20241123-kxst" src="https://github.com/user-attachments/assets/bfb0c390-b09e-46f2-b816-e9991a3c6fba">

Reply:
Ah, that’s a great way to check this. I’ll use that next time.